app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -216,40 +245,90 @@ fun CardItem(definiteIndex: Int, currentOffset: Float) {
                 }
             }
     ) {
-        val cardImage: Painter =
-            if (pageOffset > 1.5) {
-                painterResource(id = R.drawable.bg_card_small)
-            } else if (pageOffset > 0.5) {
-                painterResource(id = R.drawable.bg_card_regular)
-            } else {
-                painterResource(id = R.drawable.bg_card_large)
-            }
+        val cardImage: Painter = if (pageOffset > 1.5) {
+            painterResource(id = R.drawable.bg_card_small)
+        } else if (pageOffset > 0.5) {
+            painterResource(id = R.drawable.bg_card_regular)
+        } else {
+            painterResource(id = R.drawable.bg_card_large)
+        }
 
         Image(
-            painter = cardImage,
-            contentDescription = null,
-            modifier = Modifier.fillMaxSize()
+            painter = cardImage, contentDescription = null, modifier = Modifier.fillMaxSize()
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)